src/utils/rewriteLink.ts

```diff
@@ -96,7 +96,33 @@ const rewriteLink = (
         )
     }
 
-    // absolute links
+    // absolute links to docs (unversioned)
+    if (href.match(/^https?:\/\/docs.silverstripe.org\/en\/[a-zA-Z]/)) {
+        return createElement(
+            Link,
+            {
+                to: path.join('/', 'en', version, href.replace(/^https?:\/\/docs.silverstripe.org\/en/, '')),
+                className: 'gatsby-link'
+            },
+            domToReact(children, parseOptions)
+        );
+    }
```

Comment:
This is needed to rewrite the linkfield docs correctly, with the current version of docs applied. It won't matter immediately, but it _will_ matter when CMS 6 comes around. May as well get it right now instead.